### DISCUSSION TITLE
vincenti distance calculations

### DISCUSSION THREAD

**Discussion by morganherlocker** [Turfjs/turf#2779] [Feature proposals]
While it is a bit slower, it will still be crazy fast. Also it will be more accurate and allow for iterative calculations for linear referencing and routing stuff.

**Comment by antoniolocandro** [comment]
Maybe this document can help http://www.faa.gov/documentLibrary/media/Order/8260.54A.pdf algorithms and test cases at the end

Also as reference
https://github.com/pkohut/GeoFormulas
https://wired2code.wordpress.com/2010/07/14/wgs84-ellipsoid-calculations/
http://www.pkautomation.com/wgs84_geodesic_calc.html

**Comment by jrzimmerman** [comment]
I'm currently attempting a direct and inverse version on the vincenty formulas found here:
https://github.com/jrzimmerman/turf-vincenty-direct
https://github.com/jrzimmerman/turf-vincenty-inverse
I'll take all the help I can get! :smile: Sorry for the rookie attempt at helping, in advance

**Comment by tmcw** [comment]
@jrzimmerman added you to the turfjs team so you can work directly in the turf org!
